### PR TITLE
add: strict command feature

### PR DESCRIPTION
**Pull request by Eomm**
Hi,
I was using this lib, but I found that in some cases the Levenshtein distance is not so helpful: I have set a `restore` command that is trigger when I write `remove`.

This PR would like to add the possibility to choose if some commands need to be strict equals without using `leven`.

I would appreciate any feedback

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -84,8 +90,9 @@ function commist() {
   }
 }
 
-function Command(string, func) {
+function Command(string, strict, func) {
```
Comment (mcollina) [inline index.js]: I think the inputs here could become an option object

File: README.md [hunk 2]
```diff
@@ -45,6 +48,15 @@ Moreover, little spelling mistakes are corrected too:
 node example.js abcs cod
 ```
 
+If you want that the command must be strict equals, you can register the
+command with the json configuration:
+
+```js
+  program.register({ command: 'restore', equals: true }, function(args) {
```
Comment (mcollina) [inline README.md]: I woulc call the option strict instead.

File: index.js [hunk 3]
```diff
@@ -64,15 +64,21 @@ function commist() {
     return args
   }
 
-  function register(command, func) {
+  function register(inputCommand, func) {
+    var command = inputCommand
+    var strict = false
+    if(typeof command === 'object'){
```
Comment (mcollina) [inline index.js]: Can you add a space between if and parentheses?

It seems this is missing linting (lol!)

File: index.js (hunk 1 above)
Comment (Eomm) [reply index.js]: Done 👍

File: index.js (hunk 3 above)
Comment (Eomm) [reply index.js]: Added `standard` to the project

**Comment by Eomm** [comment]
Fixed the comments: I was careful to don't use new Node.js feature in order to let this module run with old versions.

Maybe I could add some test across node version, but I need a little help to understand how to do it

**Comment by Eomm** [comment]
Hi @mcollina,
could you tell me a release date for this feature? or there is something I can do to help you to build a release?
Thank you very much

**Pull request merged**